save-cloud-charts/save-cloud/templates/demo-service.yaml

```diff
@@ -10,3 +10,26 @@ spec:
     {{- include "service.common.ports" (dict "service" .Values.demo) | nindent 4 }}
   selector:
     {{- include "service.common.selectors" (dict "service" .Values.demo) | nindent 4 }}
+
+---
+
+# Required for debug
+
+{{ if eq .Values.demo.profile "dev" }}
```

Comment:
port forwarding doesn't fit this request?

Reply:
`port-forwarding` exposes port for a single machine while `NodePort` configures the port for every machine. Moreover, `port-forwarding` seems to configure cluster dynamically while `NodePort` does it statically.

Well, it seems that `port-forwarding` fits the request. In this case, this PR is not needed. Forgot about `port-forwarding`.